spec/mailers/recipient_selector_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe RecipientSelector do
+  subject(:recipient_selector) { Class.new { extend RecipientSelector } }
+
+  describe '.complete_registration_recipients' do
+    let!(:user_1) { create(:user, confirmed_at: 4.weeks.ago) }
+    let!(:user_2) { create(:user, confirmed_at: 4.weeks.ago) }
+    let!(:user_3) { create(:user, confirmed_at: 4.weeks.ago) }
+
+    before do
+      create(:user, :registered, confirmed_at: 1.month.ago)
+      create(:user, confirmed_at: 2.months.ago)
+    end
+
+    context 'when users are a month old but have not completed registration' do
```

Comment:
"context" is "describe" but with the expectation there'll be a corresponding alternative context. This could just be:

```ruby
it 'selects incomplete registrations in the last month' do 
```

What's the benefit of the user scopes in a module? Is it re-usability, testing? If so why not chain the scopes in `User`, test in the model, and not introduce a new object. The job will be just as dry in my opinion.